app/src/main/java/com/example/a310_rondayview/CreateEventFragment.java

```diff
@@ -22,14 +30,33 @@
 public class CreateEventFragment extends Fragment {
 
     ActivityResultLauncher<String> selectPhoto;
+    private Uri localImageUri;
+    private Uri downloadImageUri;
+    Date date;
 
     private class ViewHolder {
```

Comment:
could make this ViewHolder a static inner class -->  static inner class does not hold an implicit reference to the outer class which can help prevent memory leaks, but this is like very very nitpicky

Reply:
thanks good suggestion :cat: